stdlib/public/Concurrency/TaskPrivate.h

```diff
@@ -169,7 +169,7 @@ class TaskFutureWaitAsyncContext : public AsyncContext {
 /// The current state of a task's status records.
 class alignas(sizeof(void*) * 2) ActiveTaskStatus {
   enum : uintptr_t {
-    /// The current running priority of the task.
+    /// The max priority of the task. This is always >= basePriority in the task
```

Comment:
I don't understand how "max" is clearer than "current running".  You could say "current (escalated) priority" if you want to make the contrast more explicit.